brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
```

Comment:
It isn't. An assert will still crash in release builds, don't you prefer an exception?